docker/build-log.ps1

```diff
@@ -1,5 +1,5 @@
 Write-Host "Docker cleaning: remove old containers" -ForegroundColor Blue
-docker-compose.exe down -v --remove-orphans
+docker compose down -v --remove-orphans
```

Comment:
Did you check if the `.exe` is not needed any more ?

Reply:
Honestly, I'm not sure. It should have never been necessary. I can't get the server to respond in pure Windows using `docker-compose.exe` or `docker compose` The containers spin up, but I don't get a response from the server. I was hoping someone who uses it could provide some feedback.

I develop with Ubuntu and WSL Ubuntu. I tested the docker setup with both and it works fine.

Reply:
I saw you asked for feedback :+1: I think it's a minor use case so we should be good to merge as it is. If some run into any issue we can revisit. I'll wait a few more days for feedback before merging.

Reply:
Thanks. It is easy enough to revert if something comes up, so it should be relatively safe.